postgres-types/src/type_gen.rs

```diff
@@ -3,15 +3,15 @@ use std::sync::Arc;
 
 use crate::{Kind, Oid, Type};
 
-#[derive(PartialEq, Eq, Debug)]
+#[derive(PartialEq, Eq, Debug, Hash)]
```

Comment:
This file is generated by the code over in the codegen crate, so you'll want to go over and update that as well.